processor/src/test/java/org/mapstruct/ap/test/bugs/_2421/Issue2421Test.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.bugs._2421;
+
+import org.mapstruct.ap.testutil.ProcessorTest;
+import org.mapstruct.ap.testutil.WithClasses;
+
+public class Issue2421Test {
```

Comment:
Sorry, we would need to change the issue number to 3617 - might aswell add @IssueKey("3617") here.